### PR TITLE
[Hub Generated] Review request for Microsoft.Security to add version preview/2020-08-06-preview

### PR DESCRIPTION
**Pull request by liranc**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/liranc/azure-rest-api-specs/onPremiseSensor-dev-security-Microsoft.Security-2020-08-06-preview...master/).

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “**WaitForARMFeedback**” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/onPremiseIotSensors.json [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/OnPremiseIotSensorName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Downloaded activation file"
+          },
```
Comment (weidongxu-microsoft) [inline specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/onPremiseIotSensors.json:218]: What is the schema of the response? I assume as a "download" there had to be something in response.

File: specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/onPremiseIotSensors.json [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Updated on-premise IoT sensor",
+            "schema": {
+              "$ref": "#/definitions/OnPremiseIotSensor"
+            }
+          },
+          "201": {
```
Comment (weidongxu-microsoft) [inline specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/onPremiseIotSensors.json:140]: Is this long-running operation?

Comment (liranc) [reply specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/onPremiseIotSensors.json:140]: No, it's a synchronous operation that completes immediately.

File: specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/onPremiseIotSensors.json (hunk 1 above)
Comment (liranc) [reply specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/onPremiseIotSensors.json:218]: Actual file download (binary).

Comment (weidongxu-microsoft) [reply specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/onPremiseIotSensors.json:218]: But you need to specify it somehow? Else how SDK knows that there is a binary file as reponse?

Usually it is 
```
            "schema": {
              "type": "file"
            }
```

Comment (liranc) [reply specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/onPremiseIotSensors.json]: Nice! Pushed fix.

**Review by pilor — APPROVED** [review]

**Comment by liranc** [comment]
@pilor @weidongxu-microsoft can you please merge the PR? (the failed SDK Go check isn't related to these changes [also fails in the `master` branch]).

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**